woocommerce/woocommerce
# remove large font declaration on group product prices

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR is a companion to https://github.com/woocommerce/woocommerce-blocks/pull/9988 which removes the same large font specification.

Closes #38945 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install & activate this [Blocks branch](https://wcblocks.wpcomstaging.com/wp-content/uploads/woocommerce-gutenberg-products-block-9988.zip)
2. Switch to Twenty Twenty-Three
3. Visit a grouped product in `trunk` and this branch.
4. The prices of the products should match those in the linked Blocks PR.
5. The grouped product price font size in this PR should match the simple product price in trunk
